cmd/root_command.go

```diff
@@ -359,11 +368,6 @@ var (
 				printLoadedWebsockets(config.WebsocketConfigs)
 			}
 
-			if len(config.ValidationAllowList) > 0 {
```

Comment:
Wait why, did you remove this?

Reply:
It's a duplicated code fragment from line 372. 

Must have been a copy-paste or merge mistake I made.